### ISSUE TITLE
Export empty `theme.json` if Design Kit is in default state.

### ISSUE THREAD

**Issue by robinpalicka** [mui/mui-design-kits#428]
### Search keywords

default theme

### Latest version

- [X] I have tested the latest version

### Summary

Material UI Sync plugin generates 3333 lines long `theme.json` file even if you generate it from default MUI Kit. As a developer, you are loosing ability to see what's overridden on first glance, if this generated `theme.json` file is used.

Ideally, plugin should generate empty `theme.json`, when Design Kit is in default and untouched state. Only if some variables or e.g. Button master component is changed, then it should generate overrides for modified definitions. 

### Examples

Plugin version: `v0.1 BETA`
Design Kit version: `MUI for Figma v6.1.0 - Material UI`

Generated file from untouched Design Kit: [theme.json](https://github.com/user-attachments/files/18041149/theme.json)

**Comment by SaavanFreed** [comment]
Agreed. Why is it the Sync Plugin not just generating a diff from the default theme object..?

**Comment by oliviertassinari** [comment]
We have a default set of values that we need to update each time we do a new release of the figma kit: https://github.com/mui/mui-private/tree/master/packages/material-ui-sync-figma/src/defaults. We didn't hence, the plugin is unusable right now.
